lib/MDButtonWithSpinner.js

```diff
@@ -45,9 +45,7 @@ var MDButtonWithSpinner = function MDButtonWithSpinner(props) {
     content = _react2.default.createElement(
       _mdbutton2.default,
       buttonProps,
-      ' ',
-      children,
-      ' '
+      children
```

Comment:
@ZhengHe-MD good thing about linting is that it catches something you never expected